api/python/quilt3/packages.py

```diff
@@ -569,6 +579,7 @@ def load(cls, readable_file):
 
         return pkg
 
+    @ApiTelemetry("package.set_dir")
```

Comment:
For now let's not instrument the `set*` methods. We would benefit from visibility into this but until we have some kind of event coalescing it could blow up analytics.